include/mbedtls/config_psa.h

```diff
@@ -40,6 +40,20 @@ extern "C" {
 
 #if defined(MBEDTLS_PSA_CRYPTO_CONFIG)
 
+#if defined(PSA_WANT_ALG_CCM)
+#if !defined(MBEDTLS_PSA_ACCEL_ALG_CCM)
```

Comment:
We also want to declare these if `PSA_HAVE_SOFT_KEY_TYPE_AES` or `PSA_HAVE_SOFT_KEY_TYPE_CAMELLIA` is defined, so please move GCM and CCM after `PSA_WANT_ALG_CBC_PKCS7`, and the appropriate conditionals to this ifdef.

We also want to declare `MBEDTLS_PSA_BUILTIN_KEY_TYPE_AES`, `MBEDTLS_AES_C`, `MBEDTLS_PSA_BUILTIN_KEY_TYPE_CAMELLIA` and `MBEDTLS_CAMELLIA_C` if `(defined(PSA_WANT_ALG_GCM)
&& !defined(MBEDTLS_PSA_ACCEL_ALG_GCM)) || (defined(PSA_WANT_ALG_CCM)
&& !defined(MBEDTLS_PSA_ACCEL_ALG_CCM))` is true, so please add these conditions after `PSA_HAVE_SOFT_BLOCK_MODE ` in their declaration ifdefs.